mediawiki/LocalSettings.php

```diff
@@ -82,10 +82,14 @@
 
 ## To enable image uploads, make sure the 'images' directory
 ## is writable, then set this to true:
-$wgTmpDirectory = "$IP/images/temp";
+// $wgTmpDirectory = "$IP/images/temp";
```

Comment:
Why is this commented out? Don't we need this?

Reply:
So, it's not _required_, but this was not the problem.

When I was first working on this issue, one thing that kept coming up was that the temporary directory should be writable. So, I removed this line to let Mediawiki use it's default. I will put this line back in, test and merge this PR. Thanks for catching this! :slightly_smiling_face: